packages/utils/src/helpers/helpers.ts

```diff
@@ -65,3 +66,13 @@ export const getRdnsFromUrl = (url: string): string => {
     return ""
   }
 }
+
+export const percentageDifference = (
+  a: string | bigint,
+  b: string | bigint,
```

Comment:
I would move this to helpers/math

Maybe allow Big as arguments, since the return type is also Big.


```suggestion
  a: BigSource | bigint,
  b: BigSource | bigint,
```